configure.ac

```diff
@@ -109,6 +109,10 @@ AC_ARG_ENABLE(opus, AS_HELP_STRING([--enable-opus],
               [Build opus(audio codec) (default: no)]),
               [], [enable_opus=no])
 AM_CONDITIONAL(XRDP_OPUS, [test x$enable_opus = xyes])
+AC_ARG_ENABLE(loginfirst, AS_HELP_STRING([--enable-loginfirst],
+              [Build with login window first and session selection following (default: no)]),
+              [], [enable_loginfirst=no])
+AM_CONDITIONAL(XRDP_LOGINFIRST, [test x$enable_loginfirst = xyes])
```

Comment:
"Login first" doesn't capture the idea of the new feature. I think the session selector is more clear to the user. Users always login first.